### ISSUE TITLE
tests

### ISSUE THREAD

**Issue by hammondm** [speechbrain/speechbrain#583]
Hi.

I've been enjoying the system so far quite a bit, nice work.

I've run a few things on my own laptop+GPU, but to dive deeper, I put it all in a docker image (hammond3/speechbrain) that I could transfer over to our HPC cluster with singularity (with cuda 10.1).

I notice that one of the tests fails in that mode. `test_tokenizer`. The issue seems simple enough. The test presumes that you run the test from within the `speechbrain` directory and hardcodes the location of `dev_clean.csv`.

Easy to avoid if you're running in docker; a little harder to do with singularity.

mike h.

**Comment by TParcollet** [comment]
@pplantinga might also know

**Comment by aheba** [comment]
hi @hammondm, thanks reporting the issue, let me do a fix.

**Comment by mravanelli** [comment]
@aheba any news of that?

**Comment by aheba** [comment]
Hi @hammondm ,
Could you please share with us the error ?

I will fix it by adding `os.path.abspath` which may solve the error..

**Comment by aheba** [comment]
I made the fix on #699
Please feel free to share with us if the error still happen from you side. from mine:
```
aheba@koios:~/test_gpu/speechbrain-abdel$ pytest tests/unittests/test_tokenizer.py
============================================================================================================================= test session starts ==============================================================================================================================
platform linux -- Python 3.8.8, pytest-5.4.1, py-1.10.0, pluggy-0.13.1
rootdir: /home/aheba/test_gpu/speechbrain-abdel, inifile: pytest.ini
collected 1 item

tests/unittests/test_tokenizer.py .                                                                                                                                                                                                                                      [100%]

============================================================================================================================== 1 passed in 0.31s ===============================================================================================================================
```
